.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,6 +10,17 @@ Fixes #
 
 **Special notes for your reviewer**:
 
+**Checklist**
+
+This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
+Approvers are expected to review this list.
+
+- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
+- [ ] Maintenance: The [Boy Scout Rule](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html) was considered and applied if needed
+- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least one e2e test
+- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required
+- [ ] Community: Announce to the [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered
+
```

Comment:
```suggestion
- [ ] Upgrades: Does this change affect upgrades?
```

Or something that refers to changes potentially breaking upgrades.

Reply:
+1

Reply:
Good point. IIUC then that PR should also be marked with the label `release-note-action-required` and the release note should contain `action required` keywords.

See https://prow.ci.kubevirt.io/command-help#release_note_edit
> PRs that contain 'action required' in their 'release-note' block are given the label 'release-note-action-required' to indicate that the PR introduces potentially breaking changes that necessitate user action before upgrading to the release.